R/aedseo.R

```diff
@@ -21,10 +21,14 @@
 #' Choose between "poisson," or "quasipoisson".
 #' @param na_fraction_allowed Numeric value between 0 and 1 specifying the
 #' fraction of observables in the window of size k that are allowed to be NA.
+#' @param season A numeric vector of length 2 c(start,end) with the start and
+#' end weeks of the seasons to stratify the observations by.
+#' Ex: season = c(21,20). Default is NULL.
 #'
 #' @return A `aedseo` object containing:
 #'   - 'reference_time': The time point for which the growth rate is estimated.
 #'   - 'observed': The observed value in the reference time point.
+#'   - 'season': A stratification of observables in corresponding seasons.
```

Comment:
"The" instead of "A"? To align with the descriptions of the other return elements?